### DISCUSSION TITLE
i3wm Built-in display not showing i3 session after logging in, second display loads

### DISCUSSION THREAD

**Discussion by RabidWallaby** [i3/i3#5865] [Q&A]
New to i3wm, and WM's in general.

I use KDE & SDDM for my primary setup (EndeavourOS on an Asus ROG Strix G712LV)

Yesterday, I installed i3, to learn how WM's work & configured it to start my main display (eDP-1) and my external display (HDMI-1-0) when I login, instead of just my main display.

It was working fine after I configured it but, after a reboot, only the second display will display anything I open.  The built-in display just shows the white dot in the top corner & the login screen (I haven't configured a wallpaper loader yet).  I can move the cursor around on that screen and if I have a terminal open, the cursor changes to the text box cursor...so it knows there's supposed to be something there. 

I've tried using a login script to load both displays, and I've tried doing it within the i3 config.

If I comment out my display init lines, then my built-in display will work, but the external needs to be manually enabled.

I'm assuming this isn't a display init issue and something with i3 or xrandr not wanting to redraw after login?

Here's my i3 config lines pertaining to starting the displays:

`# Start both screens
#        exec_always --no-startup-id bash ~/.config/screenlayout/monitor.sh
        exec_always --no-startup-id xrandr --output eDP-1 --auto --primary
        exec_always --no-startup-id xrandr --output HDMI-1-0 --auto --left-of eDP-1 &
        set $mon1 eDP-1
        set $mon2 HDMI-1-0`

I commented out the startup script for now, but here's what it contains:

`!#/usr/bin/fish

xrandr --output eDP-1 --auto --primary --output HDMI-1-0 --auto --left-of eDP-1 &`

Any ideas on what's going on?

![image](https://github.com/i3/i3/assets/26662430/a7d7e46e-a3db-4e87-8e77-73d92a08397f)

**EDIT - I found a janky work-around by delaying my script launches using "sh -c 'sleep 1; SCRIPT-HERE'" but I feel like there should be a better way to do this.

**Comment by orestisfl** [comment]
See https://wiki.archlinux.org/title/multihead, specifically the part of using `/etc/X11/xorg.conf.d/` configuration
